tests/S3/S3MultiRegionClientTest.php

```diff
@@ -4,7 +4,6 @@
 use Aws\CacheInterface;
 use Aws\CommandInterface;
 use Aws\Endpoint\Partition;
-use Aws\Exception\AwsException;
```

Comment:
It looks like this could was meant to alleviate the fully qualified namespaces on [line 116](https://github.com/aws/aws-sdk-php/pull/1453/files#diff-c4eeac2a3fb38627b875d5df45cbd1ccR116), [line 143](https://github.com/aws/aws-sdk-php/pull/1453/files#diff-c4eeac2a3fb38627b875d5df45cbd1ccR143), [line 295](https://github.com/aws/aws-sdk-php/pull/1453/files#diff-c4eeac2a3fb38627b875d5df45cbd1ccR295), and [line 322](https://github.com/aws/aws-sdk-php/pull/1453/files#diff-c4eeac2a3fb38627b875d5df45cbd1ccR322).

This could be `* @expectedException AwsException` instead.

Reply:
As far as I know, annotations do not get benefited by `use`s. I am wrong?

Reply:
Ran some light tests on this one, looks like the fully qualified namespace is needed in the annotation. Apologies!

Reply:
No problem :smile: